jvm-libs/generic/extensions/kotlin/src/test/kotlin/net/consensys/StringExtensionsTest.kt

```diff
@@ -27,4 +27,30 @@ class StringExtensionsTest {
     assertThat("this includes lorem ipsum".containsAny(stringList, ignoreCase = true)).isTrue()
     assertThat("this string won't match".containsAny(stringList, ignoreCase = true)).isFalse()
   }
+
+  @OptIn(ExperimentalStdlibApi::class)
```

Comment:
I didn't catch what experimental features we're using here

Reply:
It had before, then I forgot to remove, will clean it, thanks